DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/MainActivity.java

```diff
@@ -87,4 +88,22 @@ public void onItemClicked(final ListItem item)
             startActivity( adType.getIntent() );
         }
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(final Menu menu)
+    {
+        getMenuInflater().inflate( R.menu.menu_main, menu );
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(android.view.MenuItem item)
+    {
+        if ( item.getItemId() == R.id.action_mediation_debugger )
+        {
+            AppLovinSdk.getInstance( this ).showMediationDebugger();
+        }
+
+        return true;
```

Comment:
Should return `true` only if the action is consumed, otherwise, should just return `super.onOptionsItemSelected(item);` (please see [Handling click events](https://developer.android.com/guide/topics/ui/menus#RespondingOptionsMenu) for Options menu)